app/src/main/java/com/aptoide/uploader/apps/UploadManager.java

```
@@ -65,13 +150,16 @@ public void start() {
             backgroundService.disable();
           }
         }, throwable -> {
-          throw new OnErrorNotImplementedException(throwable);
+          if (throwable instanceof NoConnectivityException) {
+            Log.e(getClass().getSimpleName(), "NO INTERNET AVAILABLE!");
+          }
```

should have an else, otherwise we are swallowing the error.